docs/installation.rst

```diff
@@ -0,0 +1,24 @@
+Installation
+============
+
+Willow supports Python 2.7+ and 3.3+. It's a pure-python library with no hard
+dependencies so doesn't require a C compiler for a basic installation.
+
+Installation using ``pip``
+--------------------------
+
+.. code-block:: shell
+
+    pip install Willow==0.3b3
```

Comment:
Probably not since the release is currently ``0.4a0``. Probably best to remove the version altogether now